### PR TITLE
Main styling rough

### PR DESCRIPTION
**Pull request by ericweissman**
Here are the changes in this PR: 

- adds images for the checks, add button
- adds components for Complete and Incomplete items
- adds rough styling to main page view (using variables for quick adjustments)
-adds .reverse() on NoteArea map for correct order of notes

### CODE REVIEW

File: src/containers/NoteArea/NoteArea.js [hunk 1]
```diff
@@ -21,9 +21,8 @@ export class NoteArea extends Component {
 
     const noteCards = notes.map(note => {
       const noteItems = items.filter(item => item.noteID === note.id)
-
-      return <NoteCard {...note} noteItems={noteItems} key={note.id} />
-    })
+      return <NoteCard {...note} noteItems={noteItems} key={note.id}/>
+    }).reverse()
```
Comment (foxwellm) [inline src/containers/NoteArea/NoteArea.js:25]: nice catch

**Review by foxwellm — APPROVED** [review]

File: src/_variables.scss [hunk 2]
```diff
@@ -1,3 +1,8 @@
 $primary : #00897b;
 $primary-light : #4ebaaa;
-$primary-dark : #005b4f;
\ No newline at end of file
+$primary-dark : #005b4f;
+
+$white: #fcfcfc;
+
```
Comment (geet084) [inline src/_variables.scss:6]: I like the naming conventions on this, I definitely need to get better. way to go

File: src/components/CompleteItem/CompleteItem.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+export const CompleteItem = (props) => {
+  return (
+    <div className='completed'>
+      <img src={checked}/>
+      <p>{props.description}</p>
+    </div>
+  )
+}
+
```
Comment (geet084) [inline src/components/CompleteItem/CompleteItem.js:14]: I like how you broke that out into a component, makes it much cleaner

File: src/components/IncompleteItem/IncompleteItem.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+export const IncompleteItem = (props) => {
+  return (
+    <div className='incomplete'>
+      <img src={unchecked}/>
+      <p>{props.description}</p>
+    </div>
+  )
+}
+
```
Comment (geet084) [inline src/components/IncompleteItem/IncompleteItem.js:13]: I like how you broke that out into a component, makes it much cleaner

File: src/containers/NoteArea/NoteArea.js (hunk 1 above)
Comment (geet084) [reply src/containers/NoteArea/NoteArea.js:25]: 👍

**Review by geet084 — APPROVED** [review]

**Pull request merged**